ocaml/xapi/cli_operations.ml

```diff
@@ -2794,6 +2794,11 @@ let vm_migrate printer rpc session_id params =
              let vdi = Client.VDI.get_by_uuid rpc session_id vdi_uuid in
              let sr = Client.SR.get_by_uuid remote_rpc remote_session sr_uuid in
              vdi,sr) (read_map_params "vdi" params) in
+         
+         let vgpu_map = List.map (fun (vgpu_uuid,gpu_group_uuid) ->
+             let vdi = Client.VGPU.get_by_uuid rpc session_id vgpu_uuid in
```

Comment:
The variable-names `vdi` and `sr` are horribly confusing.
They should be `vgpu` and `gpu_group` or similar (or avoid naming them at all).